pkg/pool-aclamm/contracts/AclAmmPool.sol

```diff
@@ -39,7 +41,14 @@ contract AclAmmPool is BalancerPoolToken, PoolInfo, Version, IBasePool, BaseHook
     uint256[] private _virtualBalances;
 
     uint256 private _c;
+
     uint256 private _sqrtQ0;
+    uint256 private _initSqrtQ0;
+    uint256 private _targetQ0;
+
+    uint256 private _startChangingQ0Timestamp;
+    uint256 private _endChangingQ0Timestamp;
```

Comment:
StablePool has an AmplificationState, like this:

```
struct AmplificationState {
    uint64 startValue;
    uint64 endValue;
    uint32 startTime;
    uint32 endTime;
}
```

I believe this could resemble that struct. We don't need 3 sqrt!0, only two: startQ0 and endQ0.

When the pool is initialized, we can store the current sqrtQ0 as startQ0 and endQ0, and the endTime as the current timestamp.